lgsm/functions/command_debug.sh

```diff
@@ -114,7 +114,7 @@ fi
 if [ "${engine}" == "source" ]||[ "${engine}" == "goldsrc" ]; then
 	${executable} ${startparameters} -debug
 elif [ "${shortname}" == "arma3" ]; then
```

Comment:
missing dayz here in the elif ?

Reply:
Yes. I had this before but this part is broken anyway. Arma3 debug don't working also on master branch. :/
Without mods, its works like is now. Of course, this needs to be changed.
https://github.com/GameServerManagers/LinuxGSM/pull/3409#issuecomment-918589291